package.json

```diff
@@ -1,5 +1,7 @@
 {
-  "name": "appium-tizen-tv-driver",
+  "name": "appium-tizen-tv-driver-monorepo",
+  "version": "0.3.0",
```

Comment:
do we want to set this to 0.0.1 or something that doesn't seem so sensible?

Reply:
ah, right. this was a copy/paste job

Reply:
(this field is not actually used for or by anything)